modules/cbQuestion/actions/qactions.php

```diff
@@ -301,5 +301,31 @@ public function delMViewWF() {
 		$rdo['notify'] = $smarty->fetch('applicationmessage.tpl');
 		echo json_encode($rdo);
 	}
+
+	public function executeScript() {
+		$record = $this->checkQIDParam();
+		$db_username = $dbconfig['db_username'];
```

Comment:
don't you have to add a "global $dbconfig" here for that array to be available?

Reply:
@joebordes , I included the global variables.